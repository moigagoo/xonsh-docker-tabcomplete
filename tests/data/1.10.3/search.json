{
        "arguments": [
                "--automated",
                "--help",
                "--no-trunc",
                "-s", "--stars"
        ],
        "commands": []
}
