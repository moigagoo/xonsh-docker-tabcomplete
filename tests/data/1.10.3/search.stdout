
Usage:	docker search [OPTIONS] TERM

Search the Docker Hub for images

  --automated        Only show automated builds
  --help             Print usage
  --no-trunc         Don't truncate output
  -s, --stars        Only displays with at least x stars
